README.md

```diff
@@ -88,6 +88,8 @@ public function write_sw() {
 console.log('<?php echo $message; ?>');
 ```
 
+Add your content generators always at load time, **before WordPress `init` action**.
```

Comment:
If we wanted to avoid this constraint, we could require the users of the library to request invalidating the files when the content they're adding changes.